shared/domains/chelonia/chelonia.js

```diff
@@ -224,7 +269,7 @@ sbp('sbp/selectors/register', {
       // but after it's finished. This is used in tandem with
       // queuing the 'chelonia/private/in/handleEvent' selector, defined below.
       // This prevents handleEvent getting called with the wrong previousHEAD for an event.
-      return sbp('okTurtles.eventQueue/queueEvent', contractID, [
+      return sbp('okTurtles.eventQueue/queueEvent', `chelonia/${contractID}`, [
```

Comment:
Could we please keep what was here originally? It's simpler/cleaner and avoids some CPU cycles too.